test/mocks/redux/redux_store_mocks.dart

```diff
@@ -11,7 +11,7 @@ class FakeStore implements Store<AppState> {
       : _state = (updates == null)
             ? AppState.init()
             : AppState.init().rebuild(updates),
-        _changeController = StreamController<AppState>();
+        _changeController = StreamController<AppState>.broadcast();
 
```

Comment:
This now matches the `Store` class' `_changeController`, see: [redux.dart/store.dart at master · fluttercommunity/redux.dart](https://github.com/fluttercommunity/redux.dart/blob/master/lib/src/store.dart)